<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlyEmBodyMergeProjectDialog</class>
 <widget class="QDialog" name="FlyEmBodyMergeProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>760</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Merge Project</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="dvidServerPushButton">
       <property name="text">
        <string>Select Database</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="menuPushButton">
       <property name="text">
        <string>Menu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="testPushButton">
       <property name="text">
        <string>Test</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QTreeView" name="verionTreeView"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="lockNodePushButton">
         <property name="text">
          <string>Lock</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="branchNodePushButton">
         <property name="text">
          <string>Branch</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_20">
     <item>
      <widget class="Line" name="line_4">
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;font color=&quot;Green&quot;&gt;Data&lt;/font&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>X</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="xSpinBox">
           <property name="maximum">
            <number>999999</number>
           </property>
           <property name="value">
            <number>3000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="ySpinBox">
           <property name="maximum">
            <number>999999</number>
           </property>
           <property name="value">
            <number>6000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Z</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="zSpinBox">
           <property name="maximum">
            <number>99999</number>
           </property>
           <property name="value">
            <number>3000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="loadGrayScalePushButton">
           <property name="text">
            <string>Load </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="labelCheckBox">
           <property name="text">
            <string>Labels</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTextEdit" name="infoWidget">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_22">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_18">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Width</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="widthSpinBox">
           <property name="minimum">
            <number>64</number>
           </property>
           <property name="maximum">
            <number>2048</number>
           </property>
           <property name="value">
            <number>512</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Height</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="heightSpinBox">
           <property name="minimum">
            <number>64</number>
           </property>
           <property name="maximum">
            <number>2048</number>
           </property>
           <property name="value">
            <number>512</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Step</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="stepSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="prevSlicePushButton">
           <property name="text">
            <string>&lt;&lt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="nextSlicePushButton">
           <property name="text">
            <string>&gt;&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Move</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <property name="spacing">
            <number>1</number>
           </property>
           <property name="leftMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>1</number>
           </property>
           <item>
            <widget class="QPushButton" name="movexyDecPushButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="gui.qrc">
               <normaloff>:/images/move_upleft.png</normaloff>:/images/move_upleft.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="movexyIncPushButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="gui.qrc">
               <normaloff>:/images/move_downright.png</normaloff>:/images/move_downright.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <property name="spacing">
            <number>1</number>
           </property>
           <property name="leftMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>1</number>
           </property>
           <item>
            <widget class="QPushButton" name="movexDecPushButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="gui.qrc">
               <normaloff>:/images/go-previous.png</normaloff>:/images/go-previous.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="movexIncPushButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="gui.qrc">
               <normaloff>:/images/go-next.png</normaloff>:/images/go-next.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <property name="spacing">
            <number>1</number>
           </property>
           <property name="leftMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>1</number>
           </property>
           <item>
            <widget class="QPushButton" name="moveyDecPushButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="gui.qrc">
               <normaloff>:/images/move_up.png</normaloff>:/images/move_up.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="moveyIncPushButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="gui.qrc">
               <normaloff>:/images/move_down.png</normaloff>:/images/move_down.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="upperVerticalLayout"/>
   </item>
   <item>
    <widget class="Line" name="line_8">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="Line" name="line_2">
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#008000;&quot;&gt;Merge Operation&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_5">
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_19">
     <item>
      <widget class="QPushButton" name="mergePushButton">
       <property name="text">
        <string>Merge</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveResultPushButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="uploadResultPushButton">
       <property name="text">
        <string>Update DVID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startSplitPushButton">
       <property name="text">
        <string>Start Split</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_21">
     <item>
      <widget class="Line" name="line_6">
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;font color=&quot;Green&quot;&gt;Message&lt;/font&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_7">
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_16"/>
   </item>
   <item>
    <widget class="QTextEdit" name="outputWidget">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FlyEmBodyMergeProjectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FlyEmBodyMergeProjectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
